src/v2/Apps/Conversation/Components/InboxHeaders.tsx

```diff
@@ -65,6 +66,17 @@ export const ConversationHeader: FC<ConversationHeaderProps> = ({
   )
 }
 
+export const EmptyInboxHeader: FC<{}> = () => {
+  return (
+    // TODO: Once Palette is updated, customize border for BorderBox.
```

Comment:
Will rebase and update once https://github.com/artsy/force/pull/5790 is in, that includes https://github.com/artsy/palette/pull/708.

Reply:
I'm trying to get that merged